File: app/src/main/java/com/asmat/rolando/popularmovies/adapters/WatchLaterMoviesGridAdapter.java
package com.asmat.rolando.popularmovies.adapters;

import com.asmat.rolando.popularmovies.R;
import com.asmat.rolando.popularmovies.models.MovieAdapterOnClickHandler;

public class WatchLaterMoviesGridAdapter extends MoviesGridBaseAdapter {
    public WatchLaterMoviesGridAdapter(MovieAdapterOnClickHandler handler) {
        super(handler);
    }

    @Override
    public int getEmptyStateLayoutID() {
        return R.layout.empty_state_watch_later;
    }
}

File: app/src/main/java/com/asmat/rolando/popularmovies/ui/adapters/grid/WatchLaterMoviesGridAdapter.kt
package com.asmat.rolando.popularmovies.ui.adapters.grid

import com.asmat.rolando.popularmovies.R
import com.asmat.rolando.popularmovies.ui.adapters.MovieAdapterOnClickHandler

class WatchLaterMoviesGridAdapter(handler: MovieAdapterOnClickHandler) : MoviesGridBaseAdapter(handler) {

    override val emptyStateLayoutID: Int
        get() = R.layout.empty_state_watch_later

}